internal/gatewayapi/backendtlspolicy.go

```diff
@@ -0,0 +1,40 @@
+package gatewayapi
+
+import (
+	"sigs.k8s.io/gateway-api/apis/v1alpha2"
+)
+
+func (t *Translator) ProcessBackendTLSPolicies(backendTlsPolicies []*v1alpha2.BackendTLSPolicy, gateways []*GatewayContext) []*v1alpha2.BackendTLSPolicy {
```

Comment:
curious why is this function needed ?

Reply:
since we are updating the status in `processBackendTLSPolicy`

Reply:
actually this is for processing the ancestor refs. updated the name.